libbeat/docs/shared-feature-flags.asciidoc

```diff
@@ -22,8 +30,27 @@ You can specify the following options in the `features` section of the +{beatnam
 [float]
 ==== `fqdn`
 
-Contains configuration for the FQDN reporting feature.  When this feature is enabled, {beatname_uc} will
-report the fully-qualified domain name for the host on which it's running.
+Contains configuration for the FQDN reporting feature. When this feature is
+enabled, {beatname_uc} reports the fully-qualified domain name for the host
+on which it's running.
```

Comment:
Similar to my comment in the Agent PR - https://github.com/elastic/observability-docs/pull/2817#discussion_r1156008574 - should we mention here that the fully-qualified domain name is reported in the `host.name` field in events produced by `{beatname_uc}`?

Reply:
I've updated the text to:
> Contains configuration for the FQDN reporting feature. When this feature is
enabled, the fully-qualified domain name for the host is reported in the
`host.name` field in events produced by {beatname_uc}.